RFC-004-revising-licenses-page.md

```diff
@@ -73,6 +73,10 @@ By changing the licenses page, we aren't "undoing" any license terms for the con
 
 If someone goes to a version of the site with the current license page (via github, archive.org, etc.), all of that material is still CC BY 4.0. We don't need to spell this out on the license page, but if someone inquires that's the answer.
 
+## Open questions
+
+* Is there an license that content files inherent by virtue of being in our repo? (Such as markdown files for the blog.) If so, this may pose a problem for us that we'll either need to account for or resolve.
```

Comment:
@bwklein Here's an open question for you regarding the current repo setup

Reply:
The license as defined in LICENSE.md in the root of the project would be the terms of use for the repo.  But we can be more specific in that file and specify the licensing for files and byproducts of them in different folders of the site. Similar to this example from another project. https://gitlab.com/Nowaker/nowaker-blog/blob/master/LICENSE.md

Reply:
Thanks! Added to section